src/applications/vaos/utils/selectors.js

```diff
@@ -334,3 +334,6 @@ export const selectSystemIds = state =>
 
 export const selectIsCernerOnlyPatient = state =>
   !!selectFacilities(state)?.every(f => f.isCerner);
+
+export const selectAtleastOneCernerFacility = state =>
+  selectFacilities(state)?.filter(facility => facility.isCerner).length > 0;
```

Comment:
Since the ticket is very clear that we need to check the `is_cerner_patient` value on the response from `GET user/` I really think we should be adding that data into Redux and checking that value. That value is set on the response directly from another service https://github.com/department-of-veterans-affairs/vets-api/pull/3947/files#diff-b920cca0b50c0bcdd6933284387430d7R109

Reply:
It says you CAN do that but it doesn't say that it is a requirement.  I figured that we could leverage other selectors that were written for the same purpose.  However, I will go ahead and make that change